### ISSUE TITLE
Installing html5lib in a fresh venv fails if setuptools is not up to date

### ISSUE THREAD

**Issue by jraby** [html5lib/html5lib-python#280]
Since #263 has been merged, it is not possible to install/upgrade setuptools at the same time as html5lib.

For example this fails:

```
cat >req.txt <<EOF
setuptools>=18.5
html5lib
EOF

virtualenv testenv
testenv/bin/pip install -r req.txt

Collecting setuptools>=18.5 (from -r req.txt (line 1))
  Using cached setuptools-24.0.3-py2.py3-none-any.whl
Collecting html5lib (from -r req.txt (line 2))
  Using cached html5lib-0.999999999.tar.gz
    Complete output from command python setup.py egg_info:
    html5lib requires setuptools version 18.5 or above; please upgrade before installing (you have 18.2)

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/49/p3xmv77x6fnb2lwl5kpvmvs40000gp/T/pip-build-0vYISn/html5lib
You are using pip version 7.1.2, however version 8.1.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

**Comment by gsnedders** [comment]
Fixed by #285 now supporting old setuptools.

**Comment by hwkns** [comment]
In the future, if the library is going to depend on something that can be installed with `pip`, maybe put it in the `install_requires` list in `setup.py`?
